pkg/cluster/install.go

```diff
@@ -201,6 +201,8 @@ func (m *manager) Update(ctx context.Context) error {
 		steps.Action(m.renewMDSDCertificate),
 		steps.Action(m.updateOpenShiftSecret),
 		steps.Action(m.updateAROSecret),
+		steps.Action(m.restartAROOperatorMaster), // depends on m.updateOpenShiftSecret; the point of restarting is to pick up any changes made to the secret
+		steps.Condition(m.aroDeploymentReady, 20*time.Minute, true),
```

Comment:
I never stopped to think about it - I just used 20 minutes because it's same timing we use in other places (admin update and install).

But since this is a context where we're not doing any huge updates and it's not during cluster install times, why don't we change it to 5 minutes? Probably still longer than needed, but will allow some time in case something weird is going on.

Reply:
+1, 5 minutes sounds good.